src/averageAges.js

```diff
@@ -15,11 +15,28 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let menArray;
+
+  switch (true) {
+    case arguments.length < 2: {
+      menArray = people.filter(person => person.sex === 'm');
+      break;
+    }
+
+    default: {
+      menArray = people.filter(person => {
+        const personLiveCentury = Math.ceil(person.died / 100);
+
+        return person.sex === 'm' && personLiveCentury === century;
+      });
+    }
+  }
```

Comment:
redundant to make switch/case for two cases
it is can be ternary in filter callback